src/qt/sparkmodel.cpp

```diff
@@ -37,7 +37,12 @@ CAmount SparkModel::getMintableSparkAmount()
 {
     std::vector<std::pair<CAmount, std::vector<COutput>>> valueAndUTXO;
     {
-        LOCK2(cs_main, wallet->cs_wallet);
+        TRY_LOCK(cs_main,lock_main);
+        if (!lock_main)
+            return 0;
```

Comment:
Can't it lead to the case when a number on overview page jump from 0 to the real value and back quickly?